src/tap/db/JDBCConnection.java

```diff
@@ -3271,7 +3271,8 @@ protected boolean isColumnExisting(String schemaName, String tableName, String c
 			int cnt = 0;
 			String columnPattern = columnCaseSensitive ? columnName : null;
 			while(rsT.next()){
-				String rsSchema = nullifyIfNeeded(rsT.getString(2));
+                                int rsTSchemaIndex = dbms.equalsIgnoreCase(DBMS_MYSQL) ? 1 : 2;
```

Comment:
The same correction should be done in `isTableExisting` and `isSchemaExisting`.
Besides, because I now know that the schema information is actually stored as a catalog name, I can enable again the testing of the schema names ; I indeed disabled it just for MySQL (see commit cf721ff298be72c1bf1651692beebe5351086a52).